### DISCUSSION TITLE
Running App sample code in Chapter 9 only displays Teapot

### DISCUSSION THREAD

**Discussion by RadFad1** [leap71/PicoGK#36] [Coding for (Computational) Engineers]
After downloading the code from the link at the end of the chapter, I noticed it ran for a while but didn't display the fixture voxFixture object, but rather just the Teapot voxel output. I commented out the line under the Run() function and the code ran fine.

![Screenshot 2024-06-22 174110](https://github.com/leap71/PicoGK/assets/149115823/e89038a3-8bbf-4ae8-a26b-a78b45437fb8)

**Comment by Mech-eng-au** [comment]
I have been struggling to find out why the code would not work. This fixed the problem. Thank you so much!

**Reply by LinKayser** [reply]
I updated all examples and fixed that issue as well. All example source code can now be found in this repo I created today:

[Coding4Engineers on my GitHub](https://github.com/LinKayser/Coding4Engineers)

And a new chapter is online — enjoy.
